src/inst_partners.yaml

```diff
@@ -0,0 +1,28 @@
+Microsoft Aure:
+  Description: Azure provides build systems for Windows, OSX, and Linux
+  Site URL: https://azure.microsoft.com/en-us/services/devops/pipelines/
+  contribution type: infrastructure
+Oregon State University Open Source Lab:
+  Description: OSL provides ppc64le machines for long running jobs
+  Logo UR: https://osuosl.org/images/osllogo-print_0.png
+  Site URL: https://osuosl.org
+  contribution type: infrastructure
+Works on Arm:
+  Description: Works on Arm project provides aarch64 machines for long running jobs
+  Logo URL: https://www.worksonarm.com/wp-content/themes/worksonarm2019/assets/images/logo.png
+  Site URL: https://www.worksonarm.com/
+  contribution type: infrastructure
+IOOS:
+  Description: The U.S. Integrated Ocean Observing System
+  Developers: ocefpaf
```

Comment:
I followed @jjhelmus' lead here. I guess we can get more info on people with the GH handle than just the name ;-p